nbs/01a_losses.ipynb

```diff
@@ -62,7 +62,15 @@
     "class BaseLoss():\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.            floatify:bool=True, # Whether targ need to be converted to float
</pre><p> </p><p>Maybe shorten to "# coerce <code>targ</code> to float?" to be more concise</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/muellerzr/fastai-docment-sprint/pull/87/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>